style.css

```diff
@@ -0,0 +1,3 @@
+h1 {
+  color: aqua;
+}
```

Comment:
- [ ] Kindly use the class name given to your `h1` element as a selector instead of using `h1` element as a selector. This is for semantics reason.